src/main/webapp/WEB-INF/views/common/index.jsp

```diff
@@ -259,6 +260,39 @@
         </div>
     </div>
 
+            <div id="duplicateBookingDialog" class="modal fade" tabindex="-1" role="dialog" aria-labelledby="myLargeModalLabel">
```

Comment:
id is **duplicate**bookingDialog, there's only one duplicateBooking-request per created booking.

It appears when you've successfully created booking, and if you like, you could add it into your corporate callendar